islandora_openseadragon.module

```diff
@@ -37,13 +40,15 @@ function islandora_openseadragon_theme() {
   return array(
     'islandora_openseadragon_viewer' => array(
       'variables' => array(
-        'uri' => '',
```

Comment:
Should we attempt a deprecation?... Could hypothetically be used elsewhere in the wild. As in, automatically coerce provided `uri` values into a `tile_sources` entry if djatoka is being used?